src/odemis/acq/move.py

```diff
@@ -368,6 +368,9 @@ def getTargetPosition(self, target_pos_lbl: int) -> Dict[str, float]:
                     sem_grid1_pos = stage_md[model.MD_SAMPLE_CENTERS][POSITION_NAMES[GRID_1]]
                     fm_target_pos = self._transformFromSEMToMeteor(sem_grid1_pos)
                 elif current_position == SEM_IMAGING:
```

Comment:
Please add a comment on top of this code explaining why it's a good idea to copy the current position and update the metadata with it. (I think the rational is that this allows to remember the SEM position (tilt) and restore it when going back from FM_IMAGING to SEM_IMAGING.